Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -134,6 +134,20 @@ open class NavigationMapView: UIView {
         }
     }
     
+    /**
+     Toggles displaying alternative routes when in active navigation session.
+     
+     If enabled, view will draw actual alternative route lines on the map.
+     Default value is `true`.
+     */
+    public var showsContinuousAlternatives: Bool = true {
```

Comment:
Cool! Does tapping on the alternative route switch to it, or is it noninteractive? I think we should only show this information by default if the user can act upon it.

Reply:
Added tap detection which switches the main route